backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/BackendVmBackupsResource.java

```diff
@@ -66,4 +73,9 @@ public Backups list() {
     public VmBackupResource getBackupResource(String id) {
         return inject(new BackendVmBackupResource(id, vmId, this));
     }
+
+    private boolean withSnapshot() {
+        return ParametersHelper.getBooleanParameter(httpHeaders, uriInfo, WITH_SNAPSHOT, true, false);
```

Comment:
ah I see why you failed HybridBackup when it's not supported - but it should be different here: we better call HybridBackup (unless the client asked otherwise) so backup applications won't need to change in order to consume this

Reply:
yes, I reversed it, will be pushed in the revision